website/package.json

```diff
@@ -1,12 +1,12 @@
 {
 	"scripts": {
-		"start": "../lib/start-server.js",
-		"build": "../lib/build-files.js",
-		"publish-gh-pages": "../lib/publish-gh-pages.js",
-		"examples": "../lib/copy-examples.js",
-		"write-translations": "../lib/write-translations.js",
-		"version": "../lib/version.js",
-		"rename-version": "../lib/rename-version.js",
+		"start": "node ../lib/start-server.js",
```

Comment:
Interesting. I didn't think about this. I just read:

https://shapeshed.com/writing-cross-platform-node/#scripts-in-package-json